src/components/Header/Header.tsx

```diff
@@ -25,7 +25,7 @@ function Header({ searchQuery, searchQueryChange }: HeaderProp) {
 		[]
 	);
 
-	const searchInapplicableScreens = ["Logged In", "My Lists"];
+	const searchInapplicableScreens = useMemo(() => ["Logged In"], []);
```

Comment:
Is this because we don't support search for list name?

Reply:
The reason this is done is due to a bug I found in our Header's search bar logic.

So, as of right now, we can search through every page except for the LoggedIn page. This is what the bug looks like and how i produced it:

1. Go to a page where we _can_ have the search bar (such as AllIngredients).
2. Click on the search icon to make the search bar appear.
3. Then, use the toolbar (with the search bar still open) to navigate to LoggedIn (where we _shouldn't_ be using a search bar).

Once that's done, our search bar carried over to the LoggedIn page. So in order for us to not render the search bar there, I used a hook on the Header where we ensure that the search bar is not shown in the inapplicable screens.